angularjs-portal-frame/src/main/webapp/css/buckyless/marketplace.less

```diff
@@ -256,13 +257,19 @@
   	cursor:pointer;
   }
   
+  img{
+  		width:100%;
+  }
```

Comment:
Please consolidate this with the other selector for img in this file that is under the screenshots
```
img {
    max-width:100%;
  }
```